src/main/java/kotlinx/reflect/lite/impl/KTypeImpl.kt

```diff
@@ -79,13 +79,13 @@ internal fun KType.computeJavaType(): Type {
             if (jClass.isArray) {
                 if (jClass.componentType.isPrimitive) return jClass
 
-                val (variance, elementType) = arguments.singleOrNull()
```

Comment:
👍

Reply:
This was a necessary change to migrate from the data class, so I would've reverted it as well to minimize unnecessary changes, but given your reaction - I guess I shouldn't? :sweat_smile: 

Tell me if you want it reverted, otherwise feel free to merge

Reply:
This is a potato-potato situation, so I do not have a strong opinion here :) 
The new version seemed to be more appealing to my preferences